src/api/app/views/webui/user/_login_form.html.haml

```diff
@@ -0,0 +1,21 @@
+#loginform.form{title: "Account login"}
```

Comment:
Class should be before id: https://github.com/brigade/haml-lint/blob/master/lib/haml_lint/linter/README.md#classesbeforeids